apps/chat/src/components/Chat/ChatMessage/ChatMessageContent.tsx

```diff
@@ -230,6 +242,7 @@ export const ChatMessageContent = ({
 
     const attachments = getUserCustomContent(
       newEditableAttachments,
+      [], // TODO
```

Comment:
What is this **TODO** about? 
I think it needs to be more info.
Or removed if it is done already.